### DISCUSSION TITLE
Insert new line

### DISCUSSION THREAD

**Discussion by A-Winters** [SubtitleEdit/subtitleedit#5455] [Ideas]
When I need to add a new line I right click and choose from "insert before this line" or "insert after this line", but more often than not I need to add a line that plays at exactly the same time as the one I'm selecting, so I choose "after" then change the start time and duration manually.
Amongst the "insert line" options, would it be possible to add an "insert line with same time & duration" option, please?
It would save me loads of time and I hope it would be useful to others too.

I really appreciate the work you all do here and I wish I knew how to code so I could help.

**Comment by LeonCheung** [comment]
At the moment, you can select a line, press **Ctrl C**, **Ctrl V**, then **Delete Text**. Just a combination of basic shortcuts.

**Reply by oswallt** [reply]
When I use Ctrl+C and then Ctrl+V it does not add the line with the same start and end times, it instead sets the start time to the end time of the currently selected line.  Is there a setting to change this behavior?

**Comment by niksedk** [comment]
You can also try the "duplicate line" shortcut.

**Reply by oswallt** [reply]
That sounds like exactly what I need!  I can't find it, though.

![Shortcuts](https://user-images.githubusercontent.com/11164281/140948752-3c0e3691-2898-4d42-8f03-a9bdb03415d6.png)

**Reply by niksedk** [reply]
Ah, yes. Wait a few days... then 3.6.3 should be out.

**Reply by A-Winters** [reply]
Thank you so much! ^_^

**Comment by LeonCheung** [comment]
Before that, you can always hit **F2** to switch to source view and copy the line number and timecodes, just don't forget to use **Ctrl Shift N** to renumber everything before closing.
